internal/cmd/group_flag.go

```diff
@@ -243,7 +246,10 @@ func handleCSVFile(client *turso.Client, file, csvTableName string) (*turso.DBSe
 	}
 	defer csvFile.Close()
 
-	errs, invalid, _ := csvlint.Validate(csvFile, ',', false)
+	if len(csvSeparator) == 0 {
+		csvSeparator = ","
+	}
+	errs, invalid, _ := csvlint.Validate(csvFile, rune(csvSeparator[0]), false)
```

Comment:
Hmm, if you're going to need a single rune here, you may want to validate the separator when getting the value from the flag.

You may find an example of how to do it [here](https://github.com/tursodatabase/turso-cli/blob/main/internal/flags/expiration.go).

I suggest doing all the verification at the flag level and making the function return a rune. Then `handleCSVFile` wouldn't receive a string, but a rune as well and you wouldn't need to care about empty strings or strings with more than one character.

Reply:
yes, that seems logical, will add the validation for the flag